index.html

```diff
@@ -53,12 +55,12 @@ <h4 class="headline">LET’S CONNECT</h4>
 <section class="card-section">
   <div class="cards-container">
     <ul class="cards-list flex-center-culomn">
-      <li class="card-item flex-center-culomn">
+      <!-- <li class="card-item flex-center-culomn">
         <div class="img-container">
           <img class="card-image" src="img/Snapshoot-Portfolio1.png" alt="Portfolio Portfolio">
         </div>
         <div class="right-container">
-          <div class="list-conatiner">
+          <div class="list-container">
             <h2>Tonic</h2>
              <div class="list">
               <span class="list-item item1">Canopy</span>
```

Comment:
- [ ] Kindly remove all commented old pieces of code as they will make your project look messy. Remember if you need to review a previous version of your code, you can always use git history.